baseline.yaml

```diff
@@ -649,6 +649,30 @@ criteria:
     security_insights_value: # TODO
     scorecard_probe: # TODO
 
+      - id: OSPS-DO-15
+    maturity_level: 2
+    category: Documentation
+    criteria: |
+       The project documentation MUST include a 
+       descriptive statement about how the project
+       selects, obtains, and tracks its dependencies.
```

Comment:
Can we pivot this from "project documentation describes" to something with more teeth?  e.g.

```yaml
- id: OSPS-QA-08
  category: Quality
  criteria: |
    The project has implemented automated review of
    changes to enforce a project-selected dependency
    policy.
```

It's not clear from this criteria whether something that says "if you add a package, remember to update `package.json`, we don't check in `package-lock.json`" could meet the `maturity_level: 2` standard.  (I'd hope not, but it _is_ a descriptive statement about how the projects obtains and tracks its dependencies...)

Reply:
These might be two separate things. You can't check against a policy that you don't have, so maybe having _any_ policy is level 1 and your suggestion is level 2?